.circleci/config.yml

```diff
@@ -914,40 +918,29 @@ jobs:
           failure_message: ':thisisfine: A \`${CIRCLE_JOB}\` job on branch \`${CIRCLE_BRANCH}\` has failed! Author Email: \`${AUTHOR_EMAIL}\`'
           mentions: "${SLACK_USER_ID}"
 
-  nightly-tests:
-    docker:
-      - image: determinedai/cimg-base:stable
-    steps:
-      - checkout
-      - setup-python-venv:
-          determined-common: true
-          determined-cli: true
-          determined: true
-          determined-deploy: true
-          extra-requirements-file: "e2e_tests/tests/requirements.txt"
-          executor: determinedai/cimg-base:stable
-      - setup-aws-cluster:
-          cluster-id: nightly-$(git rev-parse --short HEAD)-${CIRCLE_BUILD_NUM}-${CIRCLE_NODE_INDEX}
-          det-version: ${CIRCLE_SHA1}
-      - run-e2e-tests:
-          mark: "nightly"
-          master-host: ${MASTER_HOST}
-      - download-aws-logs:
-          cluster-id: ${CLUSTER_ID}
-      - terminate-aws-cluster:
-          cluster-id: ${CLUSTER_ID}
-      - store_test_results:
-          path: /tmp/test-results/
-      - slack/status:
-          fail_only: True
-          failure_message: ':thisisfine: \`${CIRCLE_JOB}\` have failed!'
-          mentions: "channel"
-          channel: "ml-ag"
-
-  e2e-gpu-tests:
+  test-e2e-gpu:
```

Comment:
non-blocking: let's call it `test-e2e-aws` for that inevitable time that we make a GCP